iFixit/react-commerce
# Add mini cart

closes #172 
closes #270 


~~deploy_block 🚫 on [this question](https://github.com/iFixit/react-commerce/issues/270#issuecomment-1156275143)~~

## Changelog

- Adds [new mini cart](https://www.figma.com/file/jRG4EyQXRDMGfyLqut08KP/iFixit---Working-files?node-id=3156%3A23831)
- Add support for csrf tokens

## QA

1. Visit the Vercel PR preview (replace vercel.app with cominor.com)
2. Verify that you're able to open the mini cart
3. Head to cominor and add something to the cart
4. Verify that the products show up in the new mini cart
5. Try to checkout as a normal customer
6. Try to checkout as a pro customer
